README.md

```diff
@@ -36,28 +36,53 @@ concurrent operations, even when using a shared folder.
-   app with "Full Dropbox" access (or "App folder" access if you prefer, if
-   you're not going to be using Dropbox's sharing features to use
-   git-remote-dropbox in a multi-user setup), and generating an access token
-   for yourself.
+1. Go to the drop box [appconsole](https://www.dropbox.com/developers/apps) (may require login)
 
-3. Save your OAuth token in `~/.config/git/git-remote-dropbox.json` or
-   `~/.git-remote-dropbox.json`. The file should look something like this:
+2. Click “Create App”
 
+3. Select “Scoped Access” (you don’t have a choice)
+    
+4. Select “Full Access”.  While you think that the repos will be “app folders”, you’ll only get one and it will be problematic to share
+
+5. Name your app
+
+6. Click “Create App”.  You will now see a configuration page
+
+7. On the “Settings” tab, click “Generated access token”. You will now see an access token.  Copy this somewhere. Note that it is longer than the display so exercise care in copying _all_ of it
+
+8. On the “Permissions” tab, under “Files and folders” select files.metadata.write (which also selects .read) and files.content.read, .write. Click “Submit” at the bottom
+
+### __Local software installations__
+
+1. Prerequisites
+   1. `python` and matching `pip`
+   2. `git`
+2. Use `pip` to install `git-remote-dropbox`.  Use `which git-remote-dropbox` to make sure it’s available via `$PATH`. If not, edit `$PATH` appropriately
+3. Save the token you generated (and copied) above in either of `~/.config/git/git-remote-dropbox.json` or `~/.git-remote-dropbox.json`. The file looks like
    ```json
    {
-       "default": "xxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxx"
+      "default": "the-token-you-copied-above"
    }
    ```
 
+4. Add the manager tool with `git config —global alias.dropbox ‘!git-dropbox-manage’`
+
+### __Note about access tokens__ 
+1. The access token you have will now enable access to your entire dropbox, from any machine. It is valid until you either delete the app or click on “Generated access token” again. If you share this token with anyone, THEY will have full access to your dropbox. Not a good experience, in general
+
+2. If you have multiple dropbox accounts, this token will access only the one that was logged in when you created the dropbox app.  You can specify alternate ones in the .json files and reference them via the pathname (See Multiple Accounts, below)
+
+## Sharing
+
+The above gives you a way to create a git remote on dropbox and use it from multiple machines (that have the access token). In other words, it’s a convenient way to share a remote with your laptop and your desktop
+
+If you want to share with other people while not giving them your access token, you should explicitly share (via the website) the root of the repo with your collaborators. They should follow steps (1) and (2), above, to enable them to use dropbox as the remote site. They will not need your access token
```

Comment:
What does "steps (1) and (2)" refer to? The other users have to do the whole access token steps 1--8 and local installation steps 1--4? Or is the 1 referring to access token and 2 referring to local installation? That might be a bit confusing because those aren't numbered right now.

Reply:
Sorry, getting correct hierarchical lists in markdown is a total PITA.

(1) and (2) refer to other participants needed to create their own app (notably with their own >private< access token) as well as doing the git-remote-dropbox installation on their own machine(s).

Reply:
You can get around the automatic formatting of ordered lists by using bulleted lists and writing "Step 1." or "1:" instead of "1." and so on. But you can also pull some steps out into a separate list and give them a named heading that's easy to refer to.